d2.config.js

```diff
@@ -0,0 +1,9 @@
+const config = {
+    type: 'app',
+
```

Comment:
This needs `coreApp: true` ([docs](https://platform.dhis2.nu/#/config/d2-config-js-reference)) in order to work as a root-level core app

Also should pass `title: 'Reports'` (or something similar) so that the browser title bar, Headerbar, and app menu items show something user-friendly.

Reply:
To be compatible with the new core app override system, `name` should also be set to `reports` here (to match `dhis-web-reports`)